### PR TITLE
Added kafka flags

### PR DESCRIPTION
**Pull request by jeongkyun-oh**
## Proposed changes

- Added two more kafka flags
  - MaxMessageBytes: the max size of a message
  - RequiredAcks: how many brokers receive the published message

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: cmd/utils/flaggroup.go [hunk 1]
```diff
@@ -141,6 +141,8 @@ var FlagGroups = []FlagGroup{
 			ChainDataFetcherKafkaTopicResourceFlag,
 			ChainDataFetcherKafkaReplicasFlag,
 			ChainDataFetcherKafkaPartitionsFlag,
+			ChainDataFetcherKafkaMaxMessageBytes,
```
Comment (aidan-kwon) [inline cmd/utils/flaggroup.go:144]: Could you change `ChainDataFetcherKafkaMaxMessageBytes` to `ChainDataFetcherKafkaMaxMessageBytesFlag`?

File: datasync/chaindatafetcher/kafka/repository.go [hunk 2]
```diff
@@ -39,7 +39,7 @@ type repository struct {
 func NewRepository(config *KafkaConfig) (*repository, error) {
 	kafka, err := NewKafka(config)
 	if err != nil {
-		logger.Error("Failed to create a new Kafka structure", "err", err, "brokers", config.Brokers, "partitions", config.Partitions, "replicas")
+		logger.Error("Failed to create a new Kafka structure", "err", err, "brokers", config.Brokers, "partitions", config.Partitions, "replicas", config.Replicas, "maxMessageBytes", config.SaramaConfig.Producer.MaxMessageBytes, "requiredAcks", config.SaramaConfig.Producer.RequiredAcks)
```
Comment (aidan-kwon) [inline datasync/chaindatafetcher/kafka/repository.go:42]: Why don't you simply print `config.String()`?

File: cmd/utils/flaggroup.go (hunk 1 above)
Comment (jeongkyun-oh) [reply cmd/utils/flaggroup.go]: Reflected. 9f4dd7b64c2024ad88c91a92a7e0a3665d7c231a

File: datasync/chaindatafetcher/kafka/repository.go (hunk 2 above)
Comment (jeongkyun-oh) [reply datasync/chaindatafetcher/kafka/repository.go]: Added a string method. 9f4dd7b64c2024ad88c91a92a7e0a3665d7c231a
Thanks

**Review by aidan-kwon — DISMISSED** [review]

**Review by KimKyungup — DISMISSED** [review]

**Review by ehnuje — APPROVED** [review]

**Comment by jeongkyun-oh** [comment]
@yoomee1313 @winnie-byun Please take a look.

**Review by winnie-byun — DISMISSED** [review]

File: cmd/utils/flags.go [hunk 3]
```diff
@@ -761,14 +761,24 @@ var (
...
-	ChainDataFetcherKafkaPartitionsFlag = cli.Int64Flag{
+	ChainDataFetcherKafkaPartitionsFlag = cli.IntFlag{
 		Name:  "chaindatafetcher.kafka.partitions",
-		Usage: "Kafka broker partition number",
+		Usage: "The number of partitions in a topic",
 		Value: kafka.DefaultPartitions,
 	}
+	ChainDataFetcherKafkaMaxMessageBytesFlag = cli.Int64Flag{
+		Name:  "chaindatafetcher.kafka.max.message.bytes",
+		Usage: "Kafka producer max size of a message",
```
Comment (yoomee1313) [inline cmd/utils/flags.go:774]: This expression is a bit confusing.. Could you make it more clear? Or what about "The max message size allowed by Kafka" ?

Comment (jeongkyun-oh) [reply cmd/utils/flags.go]: Applied 62e4d31c00f701df2dd3e49071ab0f0e2a4aa1f5. Thanks.

**Review by winnie-byun — APPROVED** [review]

**Review by aidan-kwon — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Pull request merged**